pkg/natdiscovery/proto/natdiscovery.proto

```diff
@@ -54,9 +54,13 @@ message SubmarinerNatDiscoveryResponse {
   EndpointDetails sender = 3;
   EndpointDetails receiver = 4;
 
+  bool src_ip_nat_detected = 5;
```

Comment:
I tried src_IP_NAT_detected , etc...
but the protobuf compiler generated very weird things on the go side... :/